src/pages/home/sidebar/ChatLinkRow.js

```diff
@@ -51,6 +51,12 @@ const ChatLinkRow = ({
         : styles.sidebarLinkText;
     const textUnreadStyle = option.isUnread
         ? [textStyle, styles.sidebarLinkTextUnread] : [textStyle];
+    const groupAvatar2DynamicStyles = !optionIsFocused
```

Comment:
What does "dynamic" refer to in this name?

Reply:
I was thinking "the styles that change". But i removed it in my latest commit. Thanks.